hedera-mirror-rest/viewmodel/topicMessageViewModel.js

```diff
@@ -26,19 +26,26 @@ import {encodeBase64, encodeBinary, nsToSecNs} from '../utils';
  * Topic message view model
  */
 class TopicMessageViewModel {
+  // Blockstreams no longer contain runningHashVersion, default to the latest version
+  static DEFAULT_RUNNING_HASH_VERSION = 3;
+
   /**
    * Constructs topicMessage view model
    *
    * @param {TopicMessage} topicMessage
    * @param {String} messageEncoding the encoding to display the message in
    */
   constructor(topicMessage, messageEncoding) {
+    const runningHashVersion = _.isNil(topicMessage.runningHashVersion)
```

Comment:
nit, can simplify, can also make it a one-liner to where it's assigned to `this.running_hash_version`

```suggestion
    const runningHashVersion = topicMessage.runningHashVersion ?? TopicMessageViewModel.DEFAULT_RUNNING_HASH_VERSION;
```

Reply:
Excellent, thanks I have updated it.